pkgs/tools/graphics/netpbm/default.nix

```diff
@@ -40,14 +42,25 @@ stdenv.mkDerivation {
     libjpeg
     libxml2
     libtiff
+    jbigkit
   ] ++ lib.optional enableX11 libX11;
 
+
+  strictDeps = true;
+
   enableParallelBuilding = true;
 
+  # Environment variables
+  STRIPPROG = "${stdenv.lib.getBin stdenv.cc.bintools.bintools}/bin/${stdenv.cc.targetPrefix}strip";
+
   postPatch = ''
     # Install libnetpbm.so symlink to correct destination
     substituteInPlace lib/Makefile \
       --replace '/sharedlink' '/lib'
+
+    # TODO: move to config.mk in 10.91.4
```

Comment:
Does not seem to be part of 10.91.4, even though the [commit](https://sourceforge.net/p/netpbm/code/3941/) is older than the [release](https://sourceforge.net/p/netpbm/code/3962/).

Reply:
Confirmed with upstream that it is expected to be in 10.92 (~two weeks).